### PR TITLE
Department Jacket Tweaks

### PR DESCRIPTION
**Pull request by ParadoxSpace**
Adds zippers to every single jacket's open sprite. Adds NT logo to back of Security jackets. Adds Service jacket. Makes descriptions slightly funnier.

### CODE REVIEW

**Comment by ParadoxSpace** [comment]
!review

File: html/changelogs/paradoxspace-forgotten.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.  
+author: ParadoxSpace
```
Comment (Geevies) [inline html/changelogs/paradoxspace-forgotten.yml]: Did you make the sprites?

**Comment by Geevies** [comment]
Seems it's changing this as well.
image

File: html/changelogs/paradoxspace-forgotten.yml (hunk 1 above)
Comment (ParadoxSpace) [reply html/changelogs/paradoxspace-forgotten.yml]: Credit was given when the jackets were actually added.

File: html/changelogs/paradoxspace-forgotten.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - bugfix: "The Security department jacket now has an NT logo on the back."
```
Comment (NonQueueingMatt) [inline html/changelogs/paradoxspace-forgotten.yml]: It's not just the security one.

File: code/modules/clothing/suits/miscellaneous.dm [hunk 3]
```diff
@@ -587,44 +587,52 @@
...
+	desc = "A cozy jacket in engineering's colors, featuring spacious pockets you won't even use."
 	icon_state = "engi_dep_jacket"
 	item_state = "engi_dep_jacket"
 	icon_open = "engi_dep_jacket_open"
 	icon_closed = "engi_dep_jacket"
 
 /obj/item/clothing/suit/storage/toggle/supply_dep_jacket
 	name = "supply department jacket"
-	desc = "A cozy jacket in supply's colors. Show your department pride!"
+	desc = "A cozy jacket in supply's colors, perfect for holding contraband and spare ammunition."
```
Comment (Arrow768) [inline code/modules/clothing/suits/miscellaneous.dm]: Change the description, as we do not want to encourage that mindset further.

**Pull request merged**